tests/framework/artifacts.py

```diff
@@ -0,0 +1,146 @@
+# Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+"""Define classes for interacting with CI artifacts in s3."""
+
+import os
+import platform
+import re
+
+from shutil import copyfile
+from typing import List
+from os import path
+from pathlib import Path
+
+import boto3
+import botocore.client
+
+
+MICROVM_CONFIG_EXTENSION = ".json"
```

Comment:
Does any other class except `ArtifactCollection` use these definitions? Would it make more sense to either move them under `ArtifactCollection` or move the variables defined in `ArtifactCollection` inside an enum?